### PR TITLE
feat: rambo remove web idle

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Removes Idle from web, but does *not* remove it from market-service and asset-service, so we still have Idle assets market and asset data.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/5729

## Screenshots (if applicable)

- Idle DeFi opportunities are now removed, and only available as a read-only Zapper opportunity, effectively not breaking the display of Idle positions for clients with an active Idle position 🎉 

image

- Idle market data is still happy

image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5757** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5757?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5757?utm_source=stack-comment).

File: src/assets/translations/en/main.json [hunk 1]
```diff
@@ -1573,7 +1573,6 @@
           "sendReceive": "Send/receive crypto",
           "buyCrypto": "Buy crypto",
           "tradeAssets": "Trade assets",
-          "earnYield": "Earn yield on your idle assets",
```
Comment (gomesalexandre) [inline src/assets/translations/en/main.json:1576]: revert

File: src/components/Modals/NativeOnboarding/components/EasyToUse.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ import { store } from 'state/store'
 
 import EasyToUseIcon from '../easy-to-use.svg'
 
-const NativeFeatureList = ['trackBalance', 'sendReceive', 'buyCrypto', 'tradeAssets', 'earnYield']
```
[the remark this answers is not in the document]
Comment (gomesalexandre) [reply src/components/Modals/NativeOnboarding/components/EasyToUse.tsx]: 50cd769b

**Comment by twblack88** [comment]
You're going to hate me but we just got 250K deposit

**Review by kaladinlight — APPROVED** [review]

**Comment by woodenfurniture** [comment]
as discussed we're going ahead with this as users can manage their idle externally

**Pull request merged**